Changed to use the XML Feeds, as the URLs in that feed are more complete

scoggins opened the pull request:
The one line per file data feeds sometimes truncate the URLs, the XML feeds is a more reliable source.

jtschichold commented:
@scoggins thanks for the PR ! I don't have a way to test this as we don't have direct access to AUS-CERT feeds, would you be available for a short webmeeting to see this working before accepting the PR ?

Thanks!

scoggins commented:
Hi, yeah that should be fine. I am just flat out this for the next few days with some deadlines, but will be happy to do it later in the week.

jtschichold commented:
Thanks ! please just drop me a line at lmori@paloaltonetworks.com when you it is good for you, or ping me on the Slack channel (luigimori) :-)

jtschichold commented:
I have tested the PR and everything looks good, but:
- could you change the _author_ field in the prototypes to `Simon Coggins (cqu.edu.au)` ?
- maybe the _development_status_ field should be now switched to _STABLE_ ?
- there is an additional question, see the email
- when everything is done, please could you close this PR and open a new one against the _develop_ branch instead of _master_ branch ? _develop_ branch is merged into _master_ just before the releases

Thanks !

scoggins commented:
No Problems I’ll redo it against dev! It’ll take me a few days as we’re flat out at the moment.

I didn’t want to take credit for the entire thing as I didn’t write the miner, I just updated it’s config :) But if you are happy for my name to go against it, thats fine. I’ve been using it in production for the last 2 weeks so I’m happy for it to change to STABLE.

-Simon

> On 24 Mar 2017, at 8:50 pm, Luigi Mori <notifications@github.com> wrote:
> 
> I have tested the PR and everything looks good, but:
> 
> could you change the author field in the prototypes to Simon Coggins (cqu.edu.au) ?
> maybe the development_status field should be now switched to STABLE ?
> there is an additional question, see the email
> when everything is done, please could you close this PR and open a new one against the develop branch instead of master branch ? develop branch is merged into master just before the releases
> Thanks !
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/PaloAltoNetworks/minemeld-node-prototypes/pull/34#issuecomment-288990916>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AArVtc44ThuXtOed7CAMgPOetyBZqb9tks5ro5_2gaJpZM4MZBSs>.
>

jtschichold commented:
Yes, please change the _author_ and _development_status_ fields.

Thanks to your support I have been able to test all the feeds now and it works. I have also been able to improve the Miner code to recognize invalid logins and authorization issues: PaloAltoNetworks/minemeld-core#184

scoggins commented:
Closing this pull request and will open a new one on the development branch.
